OAuth/Response/ExactOnlineUserResponse.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the HWIOAuthBundle package.
+ *
+ * (c) Hardware.Info <opensource@hardware.info>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HWI\Bundle\OAuthBundle\OAuth\Response;
+
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+
+/**
+ * @author Martijn Gastkemper <martijngastkemper@gmail.com>
+ */
+class ExactOnlineUserResponse extends PathUserResponse
+{
+	/**
+   * {@inheritdoc}
+   */
+	public function setResponse( $response )
+	{
+		parent::setResponse($response);
+		$this->response = current($this->response['d']['results']);
```

Comment:
[Exact Online uses oData 2](https://developers.exactonline.com/#OData.html%3FTocPath%3DExact%2520Online%2520REST%2520API%7C_____3) as data protocol. I agree that this line is not very nice, but a proper oData 2 library is not available for PHP. I know there are developments at Exact Online to create a library for that purpose.